install_config/storage_examples/azure_blob_docker_registry_example.adoc

```diff
@@ -0,0 +1,101 @@
+[[azure-blob-docker-registry]]
+= Configuring Azure Blob Storage for Integrated Docker Registry
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap:
+
+toc::[]
+
+[[azure-blob-docker-registry-overview]]
+== Overview
+This topic reviews how to configure Azure Blob Storage for OpenShift integrated Docker Registry.
+
+[[azure-blob-docker-registry-prerequisites]]
+== Before you begin
+
+* *Create a storage container in a blob service in a storage account*: Create a storage container using Microsoft Azure Portal, Microsoft Azure CLI, or Microsoft Azure Storage Explorer. Keep a note of the *_storage account name_*, *_storage account key_* and *_container name_*.
+* *Deploy the integrated Docker registry*: link:https://docs.openshift.com/container-platform/latest/install_config/registry/deploy_registry_existing_clusters.html#deploy-registry[Deploy the registry] if it is not deployed.
+
+[[azure-blob-docker-registry-registry-config]]
+== Overriding Registry Configuration
+
+Following steps creates a new registry pod which replaces the old pod automatically.
+
+. Create a new registry configuration file called `registryconfig.yaml` and add following information:
++
+[source,yaml]
+----
+  version: 0.1
+  log:
+    level: debug
+  http:
+    addr: :5000
+  storage:
+    cache:
+      blobdescriptor: inmemory
+    delete:
+      enabled: true
+    azure: <1>
+      accountname: azureblobacc
+      accountkey:  azureblobacckey
+      container: azureblobname
+  auth:
+    openshift:
+      realm: openshift
+  middleware:
+    registry:
+      - name: openshift
+    repository:
+      - name: openshift
+        options:
+          acceptschema2: false
+          pullthrough: true
+          enforcequota: false
+          projectcachettl: 1m
+          blobrepositorycachettl: 10m
+    storage:
+      - name: openshift
+----
+<1> Replace the values for `accountname`, `acountkey`, and `container` with *_storage account name_*, *_storage account key_*, and *_storage container name_* respectively.
+
+. Run the following command to create a new registry configuration:
```

Comment:
@gaurav-nelson Did you want to take the paragraph out as @bfallonf and @ahardin-rh suggested?